### DISCUSSION TITLE
Alternative Browser Support (LibreWolf, Ungoogled Chromium, etc.)

### DISCUSSION THREAD

**Discussion by robertlong** [matrix-org/thirdroom#371] [Community Support]
Third Room officially supports the following browsers:
- Firefox (Version 104+)
- Chrome (Version 91+)
- Edge (Version 91+)
- Safari (Version 15+)

If you're using another browser we still want to help you get into Third Room, but we're a small team and can't test every browser configuration. We've created this discussion in the hope that our community can help each other out and source solutions for issues with other browsers.

Your browser must support the following features:
  - WebGL2 (Hardware acceleration should be enabled)
  - WebRTC
  - SharedArrayBuffer
  
If your browser is labeled as a more locked down version of one of the officially supported browsers, there's a good chance that they are disabling one of the above features in some way that we can't detect and present an error message. The above APIs can be used by malicious parties for increased ability to find your browser's fingerprint and track you around the internet. However, they are absolutely necessary for Third Room to function.

If at all possible, we ask that you use Third Room with one of the officially supported browsers if your preferred browser doesn't work. If the issue resolves itself, there may be an issue with your browser's configuration. If the issue still presents itself, help us out by filing a bug report [here](https://github.com/matrix-org/thirdroom/issues).

**Comment by davidak** [comment]
Here is the result of my testing: https://pad.nixnet.services/xMU7fLEfQv2-yOiNMwzjTg

Chrome does not work for me on NixOS Linux, but Chromium works great.

Feel free to add more results to the document!

**Comment by davidak** [comment]
Which mobile browsers do you (want to) support?
